packages/cashier/src/components/error-state/error-state.tsx

```diff
@@ -1,26 +1,18 @@
 import React from 'react';
 import { localize } from '@deriv/translations';
 import { EmptyState } from '@deriv/components';
-import FundsProtection from '../funds-protection';
 
 type TProps = {
     error: unknown;
 };
 
 const ErrorState: React.FC<TProps> = ({ error }) => {
-    let code = localize('UNKNOWN');
     let message = localize('Something went wrong. Please refresh the page and try again.');
 
-    if (error && typeof error === 'object' && 'code' in error && typeof error.code === 'string') {
-        code = error.code;
-    }
-
```

Comment:
Why we removing this tho ? 🤔

Reply:
The code is not being used anywhere other than line 22 which is removed.